src/main/kotlin/cloud/luxor/lbwl/flash/listener/PlayerListener.kt

```diff
@@ -37,15 +41,17 @@ class PlayerListener(private val plugin: FlashPlugin) : Listener {
             return
         }
         if (event.action != Action.PHYSICAL) return
-        if (event.clickedBlock?.type != Material.STONE_PLATE) return
+        if (event.clickedBlock?.type != Material.STONE_PRESSURE_PLATE) return
 
         // Legacy checkpoint format
         val player = event.player
         val state = event.clickedBlock!!.location.subtract(0.0, 1.0, 0.0).block.state
 
         if (state !is Furnace) return
 
-        val number = state.inventory.name!!.toInt()
+        val number = state.inventory.contents.sumOf {
+            it?.amount ?: 0
+        }
```

Comment:
Do maps have to be changed for this to work?

Reply:
yes